### ISSUE TITLE
Edit profile: English text on French pages

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#50]
URL: https://hcdev.openplus.ca/fr/user/203/edit Modifier el profil (Edit profile)
User Agent: Chrome, Firefox
Language: French

Issue Description:  Inline error message is in English on the French page. Same as in issue #43 

Steps to reproduce: Leave one of the required fields empty and the error message appears.

Expected results: Error messages should be in French. Should follow WET error handling. avoid inline error messages.

![image](https://user-images.githubusercontent.com/14280062/128755636-f2857fbc-0ca1-49bb-a819-924965638cec.png)

![image](https://user-images.githubusercontent.com/14280062/128755605-323aaa12-095e-49b6-b8cd-d57324248a5a.png)

**Comment by echouin** [comment]
Openplus: Closed item: "This is a tool tip and we are no longer using them." Related to previously closed item # 43.

**Comment by qanitah** [comment]
Still displays when there is an error.

![image](https://user-images.githubusercontent.com/14280062/136254891-5e9a6638-d2f4-476c-91cf-44a09bda0a8b.png)

![image](https://user-images.githubusercontent.com/14280062/136255002-0230673c-0c22-4260-a6b4-b9d81708602c.png)

Also, please note that clicking on "Annuler" on "Modifier le profil" takes the user to the English Dashboard.

**Comment by echouin** [comment]
@qanitah following up w/ OP.

**Comment by echouin** [comment]
@qanitah Hello, I will be marking this issue as resolved.

**From OP:** For this issue, this is coming from the user's browser. This is not from the site.

**Comment by sarasuri** [comment]
We understand these error messages are generated from the browser when leaving the required fields empty. However, when entering invalid input in those fields custom error validation is provided. For example, on the Login page,

![image](https://user-images.githubusercontent.com/12256067/138953124-62f18ec4-4842-415b-8674-c024b6889938.png)

 
We think we shouldn’t rely on the browser generated error message when leaving the required field empty for some of these reasons.

- The method used for error validation is not consistent with the custom error validation 
- error messages are not displayed in the appropriate language of the page, 
- they disappear in few seconds of focusing into the field, this could be confusing/problematic for users
- when page is zoomed, the text size doesn’t increase for the error message accordingly.

**Comment by echouin** [comment]
@sarasuri Sent to OP.

**Comment by hydegeoff** [comment]
This is not fixed, There is extra unnecessary text in the error message as shown below.  Otherwise, If you want to keep the text, we recommend it is incorporated into the rest of the error message that is directly underneath form field.
![Unnecessary error text](https://user-images.githubusercontent.com/88147013/160686226-46602172-60c8-4399-b350-54c9c2d19b37.jpg)

**Comment by hydegeoff** [comment]
Here are other error messages:
![Error shows unnecessary text](https://user-images.githubusercontent.com/88147013/160685714-c1155ea2-b069-4940-a6ed-1ef134309f9e.jpg)

**Comment by echouin** [comment]
@hydegeoff just to check, what would need to be removed to satisfy this issue?

**Comment by davidgotlieb** [comment]
The highlighted text is actually content from the page. It's always there, even when you first load the page. I think this is just a formatting issue where on the error page the style changes. It has nothing to do with error handling.

Otherwise error handling is now using correct styles and not browser directed, so recommend closing this ticket. If the text pn the page is still an issue, recommend a new ticket be created as I feel the severity is likely very different.
